core/src/main/java/com/graphhopper/routing/ch/CHAlgoFactoryDecorator.java

```diff
@@ -63,7 +60,8 @@
 
     public CHAlgoFactoryDecorator() {
         setPreparationThreads(1);
-        setCHProfilesAsStrings(Arrays.asList(getDefaultProfile()));
+        // use fastest by default
+        setCHProfilesAsStrings(Collections.singletonList("fastest"));
```

Comment:
What if there is no fastest weighting like just shortest?

Reply:
This is not about the CH profile that is selected when a request comes in. The 'default' "fastest" here simply means that if we enable CH on `GraphHopper` it will use "fastest" unless we specify some `prepare.ch.weightings`. 

Speaking of the CH profile selection at time of a request: If only shortest is prepared and we do not specify a weighting shortest will be used. 

Does this answer your question ?

Reply:
Ah, thanks!